client/db_test.go

```diff
@@ -0,0 +1,69 @@
+package client_test
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/cockroachdb/cockroach/client"
+	"github.com/cockroachdb/cockroach/server"
+)
+
+func ExampleDB() {
```

Comment:
I was suggesting creating several examples. The original one was so small and concise, which gave it real appeal. I think now it's unnecessarily murky.